cli/automation/github-actions/deployment-workflow.md

```diff
@@ -136,57 +169,199 @@ jobs:
-          terraform_version: 1.7.4
-          terraform_wrapper: false
+          tofu_version: 1.9.0
+          tofu_wrapper: false
 
       - name: List changed stacks
         id: list
         run: terramate list --changed
 
-      - name: Authenticate to Google Cloud via OIDC
+      ## Comment this step out if not using AWS
+      - name: Configure AWS credentials via OIDC
         if: steps.list.outputs.stdout
         id: auth
-        uses: google-github-actions/auth@v2
+        uses: aws-actions/configure-aws-credentials@v4
         with:
-          workload_identity_provider: 'CHANGEME: WORKLOAD IDENTITY PROVIDER ID'
-          service_account: 'CHANGEME: SERVICE ACCOUNT EMAIL'
+          aws-region: CHANGEME_AWS_REGION
+          role-to-assume: CHANGEME_IAM_ROLE_ARN
+
+      ## Uncomment this if using Google Cloud
+      # - name: Authenticate to Google Cloud
+      #   if: steps.list.outputs.stdout
+      #   id: auth
+      #   uses: google-github-actions/auth@v2
+      #   with:
+      #     workload_identity_provider: CHANGEME_WORKLOAD_IDENTITY_PROVIDER
+      #     service_account: CHANGEME_SERVICE_ACCOUNT_EMAIL
+
+      ## Uncomment this if using Microsoft Azure
+      # - name: Configure Azure credentials
+      #   if: steps.list.outputs.stdout
+      #   id: auth
+      #   uses: azure/login@v2
+      #   with:
+      #     client-id: CHANGEME_AZURE_CLIENT_ID
+      #     tenant-id: CHANGEME_AZURE_TENANT_ID
+      #     subscription-id: CHANGEME_AZURE_SUBSCRIPTION_ID
+
+      - name: Run OpenTofu init on changed stacks
+        if: steps.list.outputs.stdout
+        id: init
+        run: |
+          terramate run \
+            --parallel 1 \
+            --changed \
+            -- \
+            tofu init
+
+      - name: Create OpenTofu plan on changed stacks
+        if: steps.list.outputs.stdout
+        id: plan
+        run: |
+          terramate run \
+            --parallel 5 \
+            --changed \
+            -- \
+            tofu plan -lock-timeout=5m -out out.otplan
 
-      - name: Run Terraform init on changed stacks
+      - name: Apply planned changes on changed stacks
+        if: steps.list.outputs.stdout
+        id: apply
+        run: |
+          terramate run \
+            --parallel 5 \
+            --changed \
+            --sync-deployment \
+            --tofu-plan-file=out.otplan \
+            -- \
+            tofu apply -input=false -auto-approve -lock-timeout=5m out.otplan
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+
+      - name: Run drift detection
+        if: steps.list.outputs.stdout && ! cancelled() && steps.apply.outcome != 'skipped'
+        id: drift
+        run: |
+          terramate run \
+            --parallel 5 \
+            --changed \
+            --sync-drift-status \
+            --tofu-plan-file=drift.otplan \
+            -- \
+            tofu plan -out drift.otplan -detailed-exitcode
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
```

Comment:
Maybe it makes sense to add an optional and commented reconcile step here that per default only runs on stacks that filter for `--tags reconcile`? For reference please see https://github.com/terramate-io/terramate-quickstart-aws/blob/main/.github/workflows/drift-detection.yml#L90